### PR TITLE
Wl 32

### PR DESCRIPTION
**Pull request by nagyDarius**

### CODE REVIEW

**Review by MarioRivis — CHANGES_REQUESTED** [review]
Why do we need these props in application-prod.yml?

**Comment by nagyDarius** [comment]
> Why do we need these props in application-prod.yml?

Because: 
- dialect: org.hibernate.dialect.PostgreSQLDialect is for generating a more efficient SQL for postgres
- the driver for the production database is prety clear

and we used this prod environment for integration tests as well, so that's why I added it already

File: src/intTest/kotlin/ro/cadeca/weasylearn/BasePostgreSQLContainerIT.kt [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+import org.springframework.test.context.ActiveProfiles
+import org.springframework.test.context.ContextConfiguration
+import org.springframework.test.context.junit4.SpringRunner
+import org.testcontainers.containers.PostgreSQLContainer
+
+
+@RunWith(SpringRunner::class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@ContextConfiguration(classes = [WeasylearnBeApplication::class], initializers = [BasePostgreSQLContainerIT.Initializer::class])
+@ActiveProfiles(profiles = ["prod"])
```
Comment (MarioRivis) [inline src/intTest/kotlin/ro/cadeca/weasylearn/BasePostgreSQLContainerIT.kt]: integrationTest

File: build.gradle.kts [hunk 2]
```diff
@@ -26,15 +47,30 @@ dependencies {
...
 	developmentOnly("org.springframework.boot:spring-boot-devtools")
+
 	runtimeOnly("com.h2database:h2")
-//	runtimeOnly("org.postgresql:postgresql")
+	runtimeOnly("org.postgresql:postgresql")
+
 	testImplementation("org.springframework.boot:spring-boot-starter-test") {
 		exclude(group = "org.junit.vintage", module = "junit-vintage-engine")
 	}
+	testRuntimeOnly("org.postgresql:postgresql")
```
Comment (MarioRivis) [inline build.gradle.kts]: remove

**Review by MarioRivis — CHANGES_REQUESTED** [review]

**Review by MarioRivis — APPROVED** [review]

**Pull request merged**